README.md

```diff
@@ -46,3 +46,16 @@ https://share.github.io/sharedb/
 ### Leaderboard
 
 [<img src="examples/leaderboard/demo.gif" height="436">](examples/leaderboard)
+
+## Development
+
+### Documentation
+
+The documentation is stored as Markdown files, but sometimes it can be useful to run these locally. The docs are served using [Jekyll](https://jekyllrb.com/), and require Ruby and [Bundler](https://bundler.io/).
```

Comment:
So I updated my Ruby install, but now I'm on version 3... Which fails because of [this issue](https://github.com/jekyll/jekyll/issues/8523)

I fixed it by cd-ing into the docs folder and running `bundle add webrick`

I can push the change to your branch if you want?